Replace link that directs incorrectly

mimckitt opened the pull request:
Replace the link http://aka.ms/appsvconmasrc1helper as it does not direct the customer correctly to the files. It sends you here instead: https://docs.microsoft.com/en-us/azure/azure-stack/

I found the correct download path to be https://appserviceonazurestack.blob.core.windows.net/rtm-62-2-40/AppServiceHelperScripts.zip 

Else, please provide the correct link to the customer and update the doc. 

Thanks :) 

-Micah

azurecla commented:
Hi __@mwinkle__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mwinkle). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
Sent to Nitin for review.

mimckitt committed:
Replace link that directs incorrectly 

Replace the link http://aka.ms/appsvconmasrc1helper as it does not direct the customer correctly to the files. It sends you here instead: https://docs.microsoft.com/en-us/azure/azure-stack/

I found the correct download path to be https://appserviceonazurestack.blob.core.windows.net/rtm-62-2-40/AppServiceHelperScripts.zip 

Else, please provide the correct link to the customer and update the doc. 

Thanks :) 

-Micah
